clients/java/src/main/java/io/zeebe/client/impl/ZeebeClientImpl.java

```diff
@@ -161,7 +163,13 @@ private static void configureConnectionSecurity(
   public static GatewayStub buildGatewayStub(
       ManagedChannel channel, ZeebeClientConfiguration config) {
     final CallCredentials credentials = buildCallCredentials(config);
-    return GatewayGrpc.newStub(channel).withCallCredentials(credentials);
+    final GatewayStub gatewayStub = GatewayGrpc.newStub(channel).withCallCredentials(credentials);
+    if (config.isMonitoringEnabled()) {
+      final MonitoringClientInterceptor monitoringInterceptor =
+          MonitoringClientInterceptor.create(Configuration.allMetrics());
```

Comment:
On adding the environment variable and tests for this functionality  I observed that creating two clients on the same JVM with monitoring enabled will faill because of this exception:

```java
java.lang.IllegalArgumentException: Collector already registered that provides name: grpc_client_started_total

	at io.prometheus.client.CollectorRegistry.register(CollectorRegistry.java:54)
	at io.prometheus.client.SimpleCollector$Builder.register(SimpleCollector.java:246)
	at me.dinowernli.grpc.prometheus.ClientMetrics$Factory.<init>(ClientMetrics.java:119)
	at me.dinowernli.grpc.prometheus.MonitoringClientInterceptor.create(MonitoringClientInterceptor.java:21)
	at io.zeebe.client.impl.ZeebeClientImpl.buildGatewayStub(ZeebeClientImpl.java:169)
	at io.zeebe.client.impl.ZeebeClientImpl.<init>(ZeebeClientImpl.java:82)
	at io.zeebe.client.impl.ZeebeClientImpl.<init>(ZeebeClientImpl.java:78)
	at io.zeebe.client.impl.ZeebeClientBuilderImpl.build(ZeebeClientBuilderImpl.java:236)
	at io.zeebe.client.ZeebeClientTest.shouldEnableMonitoringViaEnvironment(ZeebeClientTest.java:121)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)

```

It is probably not unlikely that this happens so we should think about how we want to handle this.